### PR TITLE
RT-567: Make Scenario Assessments OPTIONAL

### PR DESCRIPTION
**Pull request by NickFitz**
Also makes certain scenario assessment fields optional.

### CODE REVIEW

File: cypress/integration/scd_info_spec.js [hunk 1]
```diff
@@ -97,3 +97,34 @@ describe('The SCD info page', () => {
...
+    cy.injectAxe()
+  })
+  it('has no accessibility issues', () => {
+    // Due to header h1 and h3 with missing h2
+    cy.runA11y('html', {
+      rules: {
+        "heading-order": {enabled: false}
+      }
+    })
+  })
```
Comment (dev-digital) [inline cypress/integration/scd_info_spec.js:117]: Its not required as above scenario already validated this and it takes bloody ages to complete.

File: update_supply_chain_information/supply_chains/migrations/0042_auto_20210928_1446.py [hunk 2]
```diff
@@ -0,0 +1,145 @@
+# Generated by Django 3.2.5 on 2021-09-28 14:46
```
Comment (dev-digital) [inline update_supply_chain_information/supply_chains/migrations/0042_auto_20210928_1446.py:1]: Is this a merged migration ? _0042_ ?

**Review by dev-digital — COMMENTED** [review]
Minor comments

File: cypress/integration/scd_info_spec.js (hunk 1 above)
Comment (NickFitz) [reply cypress/integration/scd_info_spec.js:117]: Ah yes, I should have spotted that! Too much copy & pasting in a hurry. I'll take it out 😃

File: update_supply_chain_information/supply_chains/migrations/0042_auto_20210928_1446.py (hunk 2 above)
Comment (NickFitz) [reply update_supply_chain_information/supply_chains/migrations/0042_auto_20210928_1446.py:1]: Yes, unfortunately I'd already pushed this migration to GH before merging the upstream that also had an 0042, so I had to add yet another merge commit.

**Review by dev-digital — APPROVED** [review]

**Pull request merged**